webcomponents-lite.js

```diff
@@ -53,9 +53,9 @@
     'URL/URL.js',
     'WeakMap/WeakMap.js',
     'MutationObserver/MutationObserver.js',
+    'Template/Template.js',
```

Comment:
Modified `WebComponents/build-lite.json`. `WebComponents/build.json` didn't seem to directly reference templates. Any suggestions?